### DISCUSSION TITLE
Add an option to allow NetworkAnimator to sync parameters when Animator is disabled

### DISCUSSION THREAD

**Discussion by Charley3d** [FirstGearGames/FishNet#454] [Feature Request]
While using PuppetMaster with CSPv1, PM will disable Animator Component to perform animations itself.

In this configuration, it's impossible for Fishnet to sync animation's parameters over the Network.

It could be intersting to expose a method that allow us to sync Animation's parameters even though Animator component is disabled.

_I didn't test with CSPv2 but I guess issue still the same._

Link of the discussion is here:
https://discord.com/channels/424284635074134018/923983677980045382/1130282760196604005

**Comment by FirstGearGames** [comment]
Do we know if you can set parameters on a disabled animator or will warnings throw?

**Comment by FirstGearGames** [comment]
Added in 4.3
